WordPress/Jetpack/Resources/release_notes.txt

```diff
@@ -1,11 +1,13 @@
-We made a small but mighty change to the block editor: the “Add Block” button is more visible when you first open the editor with no blocks selected. We also removed one of three error notifications when a media or audio block fails to upload—it felt like overkill.
+* [**] Self hosted sites are not restricted by video length during media uploads [https://github.com/wordpress-mobile/WordPress-iOS/pull/18414]
```

Comment:
Items about self-hosted site should be removed from Jetpack Release Notes, right @leandroalonso ?

Reply:
@AliSoftware yep!

Reply:
```suggestion
```

Reply:
Dang! I missed that... Thank you for noticing and confirming @AliSoftware @leandroalonso